### PR TITLE
i18n(it): update outdated translations

### PR DESCRIPTION
**Pull request by itsmatteomanf**
#### What kind of changes does this PR include?

- Translated content

#### Description

Updated translations for:

- [x] [nav.ts](https://github.com/withastro/docs/blob/main/src/i18n/en/nav.ts)
- [x] [ui.ts](https://github.com/withastro/docs/blob/main/src/i18n/en/ui.ts)
- [x] [concepts/why-astro.mdx](https://github.com/withastro/docs/blob/main/src/content/docs/en/concepts/why-astro.mdx)
- [x] [getting-started.mdx](https://github.com/withastro/docs/blob/main/src/content/docs/en/getting-started.mdx)
- [x] [guides/backend/google-firebase.mdx](https://github.com/withastro/docs/blob/main/src/content/docs/en/guides/backend/google-firebase.mdx)
- [x] [install/auto.mdx](https://github.com/withastro/docs/blob/main/src/content/docs/en/install/auto.mdx)
- [x] [install/manual.mdx](https://github.com/withastro/docs/blob/main/src/content/docs/en/install/manual.mdx)
- [x] [recipes.mdx](https://github.com/withastro/docs/blob/main/src/content/docs/en/recipes.mdx)

### CODE REVIEW

**Comment by itsmatteomanf** [comment]
I can't tell what the updates for the two additional files. The only 2 changes in each file in the past months are already present.

File: src/content/docs/it/getting-started.mdx [hunk 1]
```diff
@@ -7,6 +7,11 @@ import TranslatorList from '~/components/TranslatorList.astro'
 import ContributorList from '~/components/ContributorList.astro'
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
+:::tip[Astro version 3 is here!]
```
Comment (csarnataro) [inline src/content/docs/it/getting-started.mdx:10]: Shouldn't we translate also "Astro version 3 is here"?

Comment (itsmatteomanf) [reply src/content/docs/it/getting-started.mdx:10]: Definitely! I totally missed that.

File: src/i18n/it/ui.ts [hunk 2]
```diff
@@ -116,4 +116,11 @@ export default {
 	'expressiveCode.copyButtonCopied': 'Copiato!',
 	// Backend Guides vocabulary
 	'backend.navTitle': 'Altre guide per servizi backend',
+	// Stubs vocabulary
+	'stub.title': 'Espandi questo abbozzo!',
+	'stub.subtitle': 'Questa guida è un abbozzo.',
+	'stub.description.migration':
+		'Vuoi contribuire a questa guida? Hai un articolo, video, o un’altra risorsa da condividere a proposito di questa tecnologia su Astro?',
+	'stub.description.cms': 'Ne sai di più su come usare questo CMS con Astro?',
```
[the remark this answers is not in the document]
Comment (itsmatteomanf) [reply src/i18n/it/ui.ts:124]: If that's the context, then I'd agree. But to me it doesn't translate like that. I did ponder this sentence a while...

It's definitely in the way I translated it, which makes not much sense probably, but that's how it's written.

We could ask @delucis...?

**Review by csarnataro — APPROVED** [review]
LGTM.
Regarding my last comment about "Know more about", if the context where this sentence is used will be clearer in the future, we can update it later.

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for this peeps 🚀

**Pull request merged**